ras/ras_extended.py

```diff
@@ -249,3 +249,23 @@ def test3_lscfg(self):
         if self.is_fail >= 1:
             self.fail("%s command(s) failed in lscfg tool verification"
                       % self.is_fail)
+
+    def test_locking_mechanism(self):
+        """
+        This tests database (vpd.db) locking mechanism when multiple 
+        instances of vpdupdate and lsvpd are running simultaneously. 
+        Locking mechanism prevents corruption of database file 
+        when running vpdupdate multiple instances
+        """
+
+        cmd = "for i in $(seq 500) ; do vpdupdate & done ;"
+        ret = process.run(cmd, ignore_bg_processes=True, ignore_status=True,
```

Comment:
@shirishaganta1 can you incorporate the review comments(from the previous PR) from Naresh?
for _ in range(500):                                                                                                                                                                    
    ret = process.run("vpdupdate", ignore_status=True, shell=True)

Reply:
For some reason the above code does not work as expected. Let's go ahead with the original code for now. We can figure out later if any additional modifications are required.